src/scripts/main.js

```diff
@@ -1 +1,57 @@
 'use strict';
+
+const firstPromise = new Promise((resolve, reject) => {
+  setTimeout(() => reject(new Error(`First promise was rejected`)), 3000);
+
+  document.addEventListener('click', () => {
+    resolve('First promise was resolved');
+  });
+});
+
+const success = (message) => {
+  const div = document.createElement('div');
+
+  div.setAttribute('data-qa', 'notification');
+  div.classList.add('success');
+  div.textContent = message;
+  document.body.append(div);
+};
+
+const error = (message) => {
+  const div = document.createElement('div');
+
+  div.setAttribute('data-qa', 'notification');
+  div.classList.add('error');
+  div.textContent = message;
+  document.body.append(div);
+};
+
+firstPromise.then(success).catch(error);
+
+const secondPromise = new Promise((resolve, reject) => {
+  document.addEventListener('click', () => {
+    resolve('Second promise was resolved');
+  });
+
+  document.addEventListener('contextmenu', () => {
+    resolve('Second promise was resolved');
+  });
+});
```

Comment:
The `secondPromise` resolves on either a 'click' or 'contextmenu' event. However, both event listeners resolve the promise with the same message. Once a promise is resolved, it cannot be resolved again, so the second event listener is redundant. Consider whether you need both events to resolve the promise or if they should trigger different actions.